2.2-TapTweak.ipynb

```diff
@@ -302,20 +302,20 @@
    "outputs": [],
    "source": [
     "# Example key pair\n",
-    "privkey = ECKey().set(102118636618570133408735518698955378316807974995033705330357303547139065928052, True)\n",
+    "privatekey = ECKey().set(102118636618570133408735518698955378316807974995033705330357303547139065928052, True)\n",
```

Comment:
Can we please just standardise on pubkey/privkey rather than have different variable names everywhere?